src/Core/ContentManager/Permalink/PermalinkGenerator.php

```diff
@@ -226,7 +226,7 @@ private function getTitleSlugified(ItemInterface $item)
         $preservePathTitle = $this->getPreservePathTitleAttribute($item);
 
         if ($preservePathTitle === true && isset($attributes['title_path']) === true) {
-            return (new StringWrapper($attributes['title_path']))->slug();
+            return $attributes['title_path'];
```

Comment:
In this case, if you are using certain characters such as spaces, you'll get a bad permalink.